test/integration_test/snapshot_test.go

```diff
@@ -1180,6 +1187,479 @@ func updateRetainValueTest(t *testing.T) {
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      scheduleNameTemp,
+			Namespace: namespace,
+		},
+		Spec: storkv1.VolumeSnapshotScheduleSpec{
+			Template: storkv1.VolumeSnapshotTemplateSpec{
+				Spec: crdv1.VolumeSnapshotSpec{
+					PersistentVolumeClaimName: "mysql-temp"},
+			},
+			SchedulePolicyName: policyName,
+		},
+	}
+
+	if authTokenConfigMap != "" {
+		err := addSecurityAnnotation(snapSched)
+		log.FailOnError(t, err, "Error adding annotations to interval schedule policy")
+		err = addSecurityAnnotation(snapSched1)
+		log.FailOnError(t, err, "Error adding annotations to interval schedule policy")
+	}
+
+	_, err = storkops.Instance().CreateSnapshotSchedule(snapSched)
+	log.FailOnError(t, err, "Error creating interval snapshot schedule")
+	_, err = storkops.Instance().CreateSnapshotSchedule(snapSched1)
+	log.FailOnError(t, err, "Error creating interval snapshot schedule")
+	defer func(schedule, schedule1 storkv1.VolumeSnapshotSchedule, ns string) {
+		err := storkops.Instance().DeleteSnapshotSchedule(schedule.Name, ns)
+		log.FailOnError(t, err, "Error creating interval schedule policy")
+		err = storkops.Instance().DeleteSnapshotSchedule(schedule1.Name, ns)
+		log.FailOnError(t, err, "Error creating interval schedule policy")
+
+	}(*snapSched, *snapSched1, namespace)
+
+	//////////////////////////////////////////////////////////
+	// Wait for creation of one volumesnapshot and validate
+	// the count to match the retain value.
+	/////////////////////////////////////////////////////////
+	sleepTime := time.Duration(1 * time.Minute)
+	log.InfoD("Created snapshotschedules %v and %v in namespace %v, sleeping for %v for schedule to trigger",
+		scheduleName, scheduleNameTemp, namespace, sleepTime)
+	time.Sleep(sleepTime)
+
+	snapStatuses, err := storkops.Instance().ValidateSnapshotSchedule(scheduleName,
+		namespace,
+		snapshotScheduleRetryTimeout,
+		snapshotScheduleRetryInterval)
+	log.FailOnError(t, err, "Error validating snapshot schedule")
+	Dash.VerifyFatal(t, len(snapStatuses), 1, "Should have snapshots for only one policy type")
+	log.InfoD("Validated snapshotschedule %s to have two volumesnapshots", scheduleName)
+
+	snapStatus, err := storkops.Instance().ValidateSnapshotSchedule(scheduleNameTemp,
+		namespace,
+		snapshotScheduleRetryTimeout,
+		snapshotScheduleRetryInterval)
+	log.FailOnError(t, err, "Error validating snapshot schedule")
+	Dash.VerifyFatal(t, len(snapStatus), 1, "Should have snapshots for only one policy type")
+	log.InfoD("Validated snapshotschedule %s to have two volumesnapshots", scheduleNameTemp)
+
+	//////////////////////////////////////////////////////////
+	// Find the pods using the PVC to determine the node.
+	// Post determining, stop portworx service on the node.
+	/////////////////////////////////////////////////////////
+	podLabel := make(map[string]string, 0)
+	podLabel["app"] = "mysql"
+	pvcList, err := core.Instance().GetPersistentVolumeClaims(namespace, podLabel)
```

Comment:
Can you make a common function to stop the driver on a node with input as a PVC?

Reply:
Done